packages/nest/README.md

```diff
@@ -25,11 +25,43 @@
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
 for feature flagging that works with your favorite feature flag management tool.
 
+<!-- x-hide-in-docs-end -->
+
 🧪 This SDK is experimental.
 
-#### Here's a basic example of how to use the OpenFeature NestJS API with `InMemoryProvider`.
+## Overview
+
+The OpenFeature NestJS SDK is a package that provides a NestJS wrapper for the [OpenFeature Server SDK](). It's main capabilities are:
+- Provide a NestJS global module to simplify OpenFeature configuration and usage within NestJS;
+- Supply custom parameter decorators (for controllers), that inject observables with feature flags resolution details;
+- Inject context for flag evaluation seamlessly using NestJS interceptors
+- Provide decorators for OpenFeature Client injection into NestJS services and controllers
+- TODO: map other features
+
+## 🚀 Quick start
+
+### Requirements
+
+- Node.js version 16+
+- NestJS
```

Comment:
Is there a minimum version?

Reply:
This is probably a good question. If you can't identify a specific min version, I would suggest going back at least one version (maybe 2) and see if it works, then call that the min.

Reply:
Actually we have it defined in peerDeps ("^8.0.0 || ^9.0.0 || ^10.0.0").
I think NestJS could be remove from this as it will be in the peerDeps table anyways and it seems doubled with this table.

Reply:
I think for transparency it might be worth just putting 8+ here.

The other thing is maybe we should use no upper bound? I guess that's a risk if there's some related breaking change, but on the other hand, this means we'll have to quickly expand this with every major Nest version

Reply:
I agree. It's easier to just fix if any major version breaks it (and maybe pinning the max version) til it's compatible instead of updating on every release.